framework/test/utils/node/state_store_mock.ts

```diff
@@ -71,9 +71,9 @@ export class StateStoreMock {
 				const index = this.accountData.findIndex(acc => acc.address.equals(address));
 				if (index > -1) {
 					this.accountData[index] = account;
-					return;
```

Comment:
i think this `return` was correct?

Reply:
It was throwing error for reason earlier after rebasing but now its not, i'll keep it to return